code/game/objects/items/stacks/sheets/leather.dm

```diff
@@ -188,15 +188,23 @@ GLOBAL_LIST_INIT(xeno_recipes, list ( \
 GLOBAL_LIST_INIT(leather_recipes, list ( \
 	new/datum/stack_recipe("wallet", /obj/item/storage/wallet, 1), \
 	new/datum/stack_recipe("muzzle", /obj/item/clothing/mask/muzzle, 2), \
+	new/datum/stack_recipe("saddle", /obj/item/saddle, 5), \
+	new/datum/stack_recipe("leather shoes", /obj/item/clothing/shoes/laceup, 2), \
+	new/datum/stack_recipe("cowboy boots", /obj/item/clothing/shoes/cowboy, 2), \
 	new/datum/stack_recipe("botany gloves", /obj/item/clothing/gloves/botanic_leather, 3), \
-	new/datum/stack_recipe("toolbelt", /obj/item/storage/belt/utility, 4), \
 	new/datum/stack_recipe("leather satchel", /obj/item/storage/backpack/satchel/leather, 5), \
-	new/datum/stack_recipe("bandolier", /obj/item/storage/belt/bandolier, 5), \
+	new/datum/stack_recipe("sheriff vest", /obj/item/clothing/accessory/vest_sheriff, 4), \
 	new/datum/stack_recipe("leather jacket", /obj/item/clothing/suit/jacket/leather, 7), \
-	new/datum/stack_recipe("leather shoes", /obj/item/clothing/shoes/laceup, 2), \
 	new/datum/stack_recipe("biker jacket", /obj/item/clothing/suit/jacket/leather/biker, 7), \
-	new/datum/stack_recipe("saddle", /obj/item/saddle, 5), \
-	new/datum/stack_recipe("sheriff vest", /obj/item/clothing/accessory/vest_sheriff, 4), \
+	new/datum/stack_recipe_list("belts", list( \
+		new/datum/stack_recipe("tool belt", /obj/item/storage/belt/utility, 4), \
```

Comment:
Broken indentation

Reply:
It is not, there is nesting, just like the cowboy hats and fancy boxes for cardboard are.

Reply:
i see